internal/matchers/binary.go

```diff
@@ -86,3 +88,41 @@ func Dcm(in []byte) bool {
 	return len(in) > 131 &&
 		bytes.Equal(in[128:132], []byte{0x44, 0x49, 0x43, 0x4D})
 }
+
```

Comment:
Since macho32 and macho64 have the same MIME type, I suggest to have a single matching function and a single node in the tree for both 32 and 64 bit versions. All seems right outside of this.

Reply:
If the project were to be updated to support metadata, or text based descriptions of the file type it would be nice to have the separate nodes.